src/coreclr/src/gc/gc.cpp

```diff
@@ -33472,6 +33575,16 @@ CObjectHeader* gc_heap::allocate_large_object (size_t jsize, uint32_t flags, int
     return obj;
 }
 
+CObjectHeader* gc_heap::allocate_large_object (size_t jsize, uint32_t flags, int64_t& alloc_bytes)
```

Comment:
why do we need these 2 methods at all instead of just calling allocate_uoh_object and passing in the right args?